weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/CallSummary.tsx

```diff
@@ -77,7 +81,7 @@ export const CallSummary: React.FC<{
               Usage
             </p>
             <CostTable
-              usage={span.summary.usage as {[key: string]: UsageData}}
+              usage={span.summary.usage as {[key: string]: LLMUsageSchema}}
```

Comment:
Seems weird that this is not typed coming in. I'd take a look at that

Reply:
I think it was not type since i was pulling from rawspan, but now it should be typed properly since its from traceCall, i removed the cast in the cost table pr